test_util/aws.py

```diff
@@ -424,6 +424,7 @@ class VpcCfStack(CfStack):
     @classmethod
     def create(cls, stack_name, instance_type, instance_os, instance_count,
                admin_location, key_pair_name, boto_wrapper):
+
```

Comment:
I'd be fine with keeping the style commit on this PR, that way I can still review the behavior changes in isolation. No big deal either way.